### DISCUSSION TITLE
How to open civit pagew from Extra network?

### DISCUSSION THREAD

**Discussion by Holyniwa** [vladmandic/sdnext#1817] [Q&A]
I updated to the latest version of automatic, and I love the new tab of extra networks (though I wish I could just remove the entire "generated images" middle section and make the extra network section larger), but I noticed that I don't have the network button that I can be taken to the civit page from the link. Is something messed up, or do i need to enable it somewhere? 

Here's what i looks like to me, no link buttons like previous versions
![opera_4j1QUjfl1h](https://github.com/vladmandic/automatic/assets/13576034/c05dbc26-25cf-4bb0-a8e9-74e2f66a6ea9)

**Comment by vladmandic** [comment]
civit page link was never part of core repo, it was added by extension. and those extensions may have issues creating links in the new layout.

> though I wish I could just remove the entire "generated images" middle section and make the extra network section larger

but you can, its in settings.
